pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -184,6 +184,12 @@ func (c *Cluster) checkHealth() error {
 		}
 	}
 
+	//Remove any pending/not needed mon canary deployment if everything is ok
```

Comment:
```suggestion
	// Remove any pending/not needed mon canary deployment if everything is ok
```

Reply:
thx and sorry for forget always this things .... lets see if i can put any kind of linter/tool to avoid this errors.